python/geocode.py

```diff
@@ -81,10 +84,7 @@ def parse_output(geo):
     )
 
 
-if __name__ == "__main__":
-    # connect to BUILD_ENGINE
-    engine = create_engine(os.environ["BUILD_ENGINE"])
-
+def load_applications(engine):
```

Comment:
This might be overly nit-picky but can you rename the function to load_dob_applications() and/or write a comment about what the function does - I think to us its probably more obvious but one of the things about dev db is that there are essentially two different types of applications: ones we get from dob BIS and others we get from dob now and I think by adding a comment/name it'll be more clear that this function is geocoding both bis and now